src/test/java/org/ays/auth/service/impl/AysUserUpdateServiceImplTest.java

```diff
@@ -521,4 +522,142 @@ void givenValidIdAndUserUpdateRequest_whenRolesNotExist_thenThrowAysRolesNotExis
                 .save(Mockito.any(AysUser.class));
     }
 
+
+    @Test
+    void givenValidId_whenUserIsPassive_thenActivateUser() {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
```

Comment:
Testlerde verileri daha net görebilmek adına aşağıdaki şekilde kullanabiliriz, diğer testlerdeki kullanımlar da yavaş yavaş düzeltilecek
```suggestion
        String mockId = "21a0ab5a-c0e9-4789-9704-a6b5c02e2325";
```